cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
```

Comment:
Do we need this file in the repository? If yes, should we adapt it to one example for us?

Reply:
At the moment we don't. It would require more complex tests already implemented to be used properly.

Removed this file on [6f10f0f](https://github.com/HathorNetwork/hathor-wallet/pull/365/commits/6f10f0ffa1e4397c6794058f13d7a9129768ee19) and added a section on the design's _Future Possibilities_ describing how to use this feature later, as it is a good test practice to have.